packages/frontend-2/components/settings/workspaces/General/LeaveDialog.vue

```diff
@@ -41,7 +42,9 @@ graphql(`
 `)
 
 const props = defineProps<{
-  workspace: SettingsWorkspaceGeneralDeleteDialog_WorkspaceFragment
+  workspace:
```

Comment:
I think you can leave this prop as the current fragment, and then just include `SettingsWorkspaceGeneralDeleteDialog_WorkspaceFragment` in the members table